src/skin/launchimage.h

```diff
@@ -0,0 +1,22 @@
+#ifndef LAUNCHIMAGE_H_
```

Comment:
Please document this class, including file-level description of why it exists and how it is used.